assets/images/svg-icons/circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<circle cx="8" cy="8" r="7.5" stroke="currentColor"/>
+</svg>
```

Comment:
We should keep a common 24x24 viewbox so sizing is consistent across all icons. 
We have also been converting strokes to filled paths, following Gutenberg practices, so there is less issues when scaling and coloring the icons. (Figma has an `Outline stroke` right-click action for this, but I think the replaced `lesson-status-*` SVGs were already done like that)

Reply:
Done here: https://github.com/Automattic/sensei/pull/4577/commits/695378176d1f759c62a037a3c4e4d200333b0037
I also added a readme with the standard. ;)